docs/pages/blog/mui-x-v7-beta.md

```diff
@@ -0,0 +1,247 @@
+:::
+
+### Server-Side Data Source
+
+Integration with the server side has been a huge pain point for our community; we aim to enhance developer experience and productivity by introducing the `DataSource` interface.
+
+This new interface establishes more efficient data management and seamless interactions, allowing developers to build more responsive and robust applications that connect UI elements to backend services with less complexity.
+
+It's initially designed for the Data Grid, but we plan that the DataSource will serve data to multiple components in the future.
+
+```jsx
+const myDataSource: DataSource = {
+  getRows: async (params: GetRowsParams): GetRowsResponse => {
+    // fetch data from server
+    const response = await fetch('https://my-api.com/data', {
+      method: 'GET',
+      body: JSON.stringify(params),
+    });
+    const data = await response.json();
+    // return the data and the total number of rows
+    return {
+      rows: data.rows,
+      rowCount: data.totalCount,
+    };
+  },
+};
+```
+
+```jsx
+<DataGridPro dataSource={myDataSource} columns={columns} />
+```
+
+Learn more on [the server side integration page](https://next.mui.com/x/react-data-grid/server-side-data/#data-source).
+
+### Date Object support in filter model
+
+The `filterModel` now supports `Date` objects for `date` and `dateTime` column types, providing a more intuitive and efficient filtering experience.
+
+While string values remain compatible for these types, any updates to the `filterModel` made through the UI (such as via the filter panel) will now automatically use `Date` objects, ensuring consistency and ease of data handling.
+
+### New stable features
+
+During major versions, MUI X releases new features under the `experimental` flag as a failsafe in case there's a need to change the API based on user feedback.
+
+We're now excited to announce that the following features have been promoted to stable and don't require the use of the flag anymore.
+
+- [Cell selection](https://next.mui.com/x/react-data-grid/cell-selection/) [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+- [Clipboard pasting](https://next.mui.com/x/react-data-grid/clipboard/#clipboard-paste) [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+- [Column grouping](https://next.mui.com/x/react-data-grid/column-groups/)
+
+## TreeView
+
+Following the promotion of the component from the labs to MUI X, our primary focus has been on improving its developer experience.
+This includes efforts to clarify documentation and improve key examples, making them more informative and user-friendly.
+
+A particularly significant initiative is the time-saving alternative to the traditional JSX architecture, the RichTreeView.
+
+### RichTreeView
+
+We've evolved the TreeView into two distinct components.
```

Comment:
'evolved' sounds a bit strange here 🤔 Maybe replacing it with 'restructured' or 'separated' could work?